src/components/ui/Checkbox/Checkbox.stories.mdx

```diff
@@ -0,0 +1,335 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs'
+
+import Checkbox from '.'
+
+<Meta title="Atoms/Checkbox" component={Checkbox} />
+
+export const Template = ({ id, ...args }) => (
```

Comment:
If you apply the changes I suggested [here](https://github.com/vtex-sites/gatsby.store/pull/61/files#r882836348), you also need to add `disabled` as parameter here and `<Checkbox id={id} disabled={disabled} {...args} />`

Reply:
I don't think so, Fan. We can pass the `disabled` as `{...args}`. 

We need to pass the `id` in that way because we are using the `<label htmlFor={id} ... >`.

Reply:
Oh yeah! You're right! It seems like we don't need to pass anything extra here 😀